server/server.js

```diff
@@ -42,7 +38,7 @@ app.use(
 );
 
 // Render React as View
-app.use(express.static(path.join(__dirname, "build")));
+app.use(express.static(path.join(__dirname, "..", "client", "build")));
```

Comment:
I don't think we should access the build folder this way but that's my personal opinion.